cmd/update-main.go

```diff
@@ -140,6 +141,26 @@ func IsKubernetes() bool {
 	return os.Getenv("KUBERNETES_SERVICE_HOST") != ""
 }
 
+// Minio Helm chart uses DownwardAPIFile to write pod label info to /podinfo/labels
+// More info: https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/#store-pod-fields
+// Check if this is Helm package installation and report helm chart version
+func getHelmVersion(helmInfoFilePath string) string {
+	// Check if the file exists. Ignore errors as Minio can be deployed without Helm charts as well.
+	helmInfoFile, err := os.Open(helmInfoFilePath)
+	if err == nil {
```

Comment:
Don't write it like this write it like , follow this pattern of eliminating errors first. 

```
helmInfoFile, err := os.Open(helmInfoFilePath)
if err != nil  && !os.IsNotExist(err) {
       errorIf(err, "Unable to read %s", helmInfoFilePath)
       return ""
}
... scanner 
```